lib/esbuild/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Evan Wallace
```

Comment:
Everything below here is vendor'd from `esbuild`'s internals, with the exception of some modifications to `css_printer`.